scripts/component_docs_generator/helpers/file_downloader.go

```diff
@@ -0,0 +1,117 @@
+package helpers
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+)
+
+func GetIntegrationDocsCSVFile(csvURL string) {
+
+	destinationFile := "integration.csv"
+
+	resp, err := http.Get(csvURL)
+	if err != nil {
+		log.Println("Error while fetching the CSV:", err)
+		return
+	}
+	defer resp.Body.Close()
+
+	csvContent, err := io.ReadAll(resp.Body)
```

Comment:
Use of BufferedReader might be advantageous? How large is the CSV file?

Reply:
The size is around 18Mb will have a look and make changes for this too

Reply:
Changed to BufferedReader @MUzairS15